Bots Explained

```diff
@@ -0,0 +1,34 @@
+When my Mentor Oscar Oranagwa asked me to write about bots, I decided to use this opportunity to clarify exactly on what bots
+are, to both my fellow developers as well as any newbie with the desire to benefit from this innovation. I will use a story
+to explain it all to an extent but before that, let me begin with;
+
+WHAT ARE BOTS?
+BOTS are also known as Internet Robots, and just as the name Robot implies; Bots make things easier, they can handle 
```

Comment:
`
BOTS are also known as Internet Robots, and just as the name Robot implies; Bots make things easier. They can handle ...`

Add full stop after easier.